devops/jobs/MongoAgentsUpdateCheck.groovy

```diff
@@ -0,0 +1,67 @@
+package devops.jobs
+import static org.edx.jenkins.dsl.Constants.common_logrotator
+
+class MongoAgentsUpdateCheck {
+    public static def job = {
+        dslFactory ->
+        dslFactory.job("Monitoring" + "/mongo-agents-update-check") {
+            parameters {
+                stringParam('CONFIGURATION_REPO', 'https://github.com/edx/configuration.git')
+                stringParam('CONFIGURATION_BRANCH', 'master')
+            }
+
+            multiscm {
+                git {
+                    remote {
+                        url('$CONFIGURATION_REPO')
+                        branch('$CONFIGURATION_BRANCH')
+                    }
+                    extensions {
+                        cleanAfterCheckout()
+                        pruneBranches()
+                        relativeTargetDirectory('configuration')
+                    }
+                }
+            }
+
+            triggers {
+                cron('H 7 * * *')
+            }
+
+            steps {
+                virtualenv {
+                    pythonName('System-CPython-3.5')
+                    nature("shell")
+                    systemSitePackages(false)
+
+                    command(
+                        dslFactory.readFileFromWorkspace("devops/resources/mongo-agents-update-check.sh")
+                    )
+                }
+            }
+
+            publishers {
+
+                extendedEmail {
+                    recipientList('devops@edx.org')
+                    replyToList('devops@edx.org')
```

Comment:
This is a job in the public repo.  Please don't hardcode our email address.  Pass it in from the configuration file used by other monitoring jobs.  The standard is NOTIFY_ON_FAILURE https://github.com/edx/edx-internal/blob/master/tools-edx-jenkins/monitoring.yml#L77 Feanil asks that we use something like devops+mms@edx.org to help filtering